django/utils/autoreload.py

```diff
@@ -370,6 +370,19 @@ def stop(self):
 class StatReloader(BaseReloader):
     SLEEP_TIME = 1  # Check for changes once per second.
 
+    def __init__(self):
+        self.requests_started_tracker = itertools.count()
```

Comment:
Note: I use "tracker" instead of "counter" because the number in `self.requests_started_tracker` is not necessarily equal to the number of requests started. Calling `wait_for_requests_to_finish` increments the value too. This is done because `itertools.count` does not have an interface to read the current counter value without incrementing it too. I chose to use `itertools.count` over `collections.Counter` or just `number += 1` in order to ensure that the counter is thread safe.